packages/vue/src/components/FTextField/FTextField.vue

```diff
@@ -351,6 +349,17 @@ export default defineComponent({
         onPendingValidity(): void {
             this.validityMode = "INITIAL";
         },
+        onValidationConfigUpdate(): void {
+            // technical dept (validation): for some reason nextTick is not enough
+            setTimeout(() => {
+                if (!this.inputRef) {
+                    return;
+                }
+
+                // revalidate field when config is updated
+                ValidationService.validateElement(this.inputRef);
+            }, 100);
```

Comment:
Den här funderar jag lite över, jag antar att du testat med 0 och det inte fungerade?

Inne i `validateElement(..)` kan du se om konfigurationen inte slagit igenom eller varför det inte fungerar? Känns inte riktigt bra att lägga in arbiträr fördröjning så här.

Reply:
Jag håller med, det inget vi kan merga (körde som draft just av denna anledningen).

I datumfältet funkar det fint med `await nextTick` men om man kör på samma sak här drar det med sig diverse problem i unittester. E2E-testerna för korsvalidering fungerar dock fint med `await nextTick`.